tests/ui/attributes/unix_sigpipe/unix_sigpipe-inherit.rs

```diff
@@ -1,14 +1,29 @@
+//@ ignore-cross-compile because aux-bin does not yet support it
+//@ only-unix because SIGPIPE is a unix thing
+//@ aux-bin: assert-inherit-sig_dfl.rs
+//@ aux-bin: assert-inherit-sig_ign.rs
 //@ run-pass
-//@ aux-build:sigpipe-utils.rs
 
-#![feature(unix_sigpipe)]
+#![feature(rustc_private, unix_sigpipe)]
 
-#[unix_sigpipe = "inherit"]
+extern crate libc;
+
+// By default the Rust runtime resets SIGPIPE to SIG_DFL before exec:ing child
```

Comment:
As an aside: Regarding `exec:ing`, I've never seen this typographic convention before (non-native speaker), I only know of using `'` in such contexts.

Reply:
Indeed (I'm not a native speaker either):

```console
$ git grep "'ing" | wc -l
24
```

I'll fix it in upcoming PRs in this area.